### ISSUE TITLE
Cadres d'acquisition non disponibles dans les filtres de la synthèse

### ISSUE THREAD

**Issue by MathieuLagarde** [PnX-SI/GeoNature#1841]
Bonjour,
Sur GéoNature 2.9.2, page principale de la synthèse, les "jeux de données" sont bien remontés dans le filtre concerné dans le bloc de recherche, mais pas les "cadres d'acquisition" (No items found).
Y-a-t'il une raison pour laquelle une partie des informations ne serait pas remontée (CA vs JDD) ? Comment faire remonter ces informations dans le filtre de requête ?
Par avance, merci.
Mathieu

**Comment by Splendens** [comment]
Je viens d'essayer sur plusieurs instances différentes en 2.9.2 et ça fonctionne bien. 
Tu as des erreurs dans les logs ou dans la console ?

**Comment by DonovanMaillard** [comment]
idem sur mes instances, non reproduit :)

**Comment by MathieuLagarde** [comment]
Je n'ai rien dans les logs.
Par contre, j'ai oublié de préciser que les CA manquent sous le profil "Public", mais sont bien présents sous le profil "Admin".
Une question de droit ? Et si oui, pourquoi sur les CA et pas les JDD ?

**Comment by DonovanMaillard** [comment]
c'est sur ton instance en production ? De mon coté avec le role public sur ton instance (via l'atlas) j'ai bien les CA dans mes filtres

**Comment by MathieuLagarde** [comment]
Non, justement. C'est sur 2 instances autres. Notre future prod, et une instance de recettes sur un serveur interne.
La première a été installée directement dans la dernière version, la seconde a été installée dans la version de la prod que tu as testé, puis mise à jour.

**Comment by camillemonchicourt** [comment]
A voir si il y a des infos dans la console du navigateur, mais sinon c'est certainement un problème de droits entre les métadonnees, les acteurs et le CRUVED du compte public.

**Comment by MathieuLagarde** [comment]
Je viens de comprendre mon erreur: le CRUVED du compte "public" était bon, mais celui du "Grp_public" était à 0 sur les METADATA...
C'est tout bon désormais.
Par contre, je ne comprends toujours pas pourquoi les JDD étaient remontés du coup.

**Comment by MathieuLagarde** [comment]
Bonjour,

Je complète ce ticket pour préciser un point qu'il pourrait être intéressant de corriger.

En éditant le CRUVED des métadonnées [droits Lire (R):3 et Exporter (E):3 pour l'utilisateur "public" et le groupe "grp_public"] de façon à permettre à l'utilisateur "public" de requêter les métadonnées "cadre d'acquisition" dans la page Explorer les données (synthèse), je rends également disponible le module Métadonnées à cet utilisateur, ce qui, à la base, n'était pas prévu. Si la mise à disposition du module "métadonnées" au public me convient, ce fonctionnement doit-il être néanmoins corrigé ?

Pour info, la plateforme Biodiv'Bretagne actuellement en fonctionnement est en version 2.6.2 (mais plus pour longtemps). Le CRUVED pour les métadata est à 0 pour l'utilisateur public et le grp_public. Les cadres d'acquisition sont bien requêtables en synthèse, et le module n'est pas accessible au public.

**Comment by bouttier** [comment]
Bonjour,

Depuis le module de synthèse, on propose à la recherche la liste des jeux de données accessible au regard du droit R du module synthèse. En revanche, en ce qui concerne les cadres d’acquisition, on regarde le droit R du module méta-données, ce qui oblige à élever les permissions sur ce module alors que l’on ne souhaite pas nécessairement le rendre accessible.

Pour résoudre ce problème, il faut implémenter le chargement des cadres d’acquisition disponible à la recherche en se fondant sur le R de la synthèse.

**Comment by camillemonchicourt** [comment]
@Pierre-Narcisi, fait dans la 2.13 ?

**Comment by camillemonchicourt** [comment]
En fait, ce n'est pas fait dans la 2.13, et c'est même plutôt l'inverse.
Désormais on regarde les permissions R sur le module Métadonnées pour lister les JDD et CA dans les filtres de la Synthèse, comme expliqué ici - https://github.com/PnX-SI/GeoNature/issues/2714
